explorer/src/pages/ServiceProviders/index.tsx

```diff
@@ -0,0 +1,73 @@
+import React, { useEffect, useState } from 'react';
+import { Card, FormControl, Grid, MenuItem, Select, SelectChangeEvent } from '@mui/material';
+import { Api } from '../../api';
+import { TableToolbar } from '../../components/TableToolbar';
+import { Title } from '../../components/Title';
+import { UniversalDataGrid } from '../../components/Universal-DataGrid';
+import { DirectoryService } from '../../typeDefs/explorer-api';
+
+const columns = [
+  {
+    headerName: 'Client ID',
+    field: 'address',
+    width: 1000,
+    disableColumnMenu: true,
+  },
+];
+
+const SupportedApps = () => {
+  const [selected, setSelected] = useState<string>('');
+  const handleChange = (e: SelectChangeEvent) => setSelected(e.target.value);
+  return (
+    <FormControl size="small">
+      <Select value={selected} onChange={handleChange} displayEmpty sx={{ mr: 2 }}>
+        <MenuItem value="">Supported Apps</MenuItem>
+        <MenuItem sx={{ opacity: 1 }}>Keybase</MenuItem>
+        <MenuItem>Telegram</MenuItem>
+        <MenuItem>Electrum</MenuItem>
+      </Select>
+    </FormControl>
+  );
+};
```

Comment:
Nothing happens when I click on an app on the select menu

Reply:
That's correct for the moment.